pkg/users/users.js

```diff
@@ -164,11 +164,14 @@
             />
         );
     } else if (path.length === 1) {
+        const account = accountsInfo?.find(account => account.name === path[0]);
+        if (!account)
+            return <AccountNotFound />
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-7fd6cddb-20250813-150050-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)